content/templates/profile.md

```diff
@@ -1,13 +1,20 @@
 ---
-created: 2023-11-17
-title: 
+created: 2023-11-30
+contributors: 
+title: title of the wiki entry, e.g. name of organisation, person, or concept
 url: 
-city: 
-country: 
+locations: 
+started: 
```

Comment:
'start' not started

Reply:
ambivalent... will it be a lot of work to change all the old profiles from started to start or can we do that quickly in bulk somehow ?

for now, switched to start and end instead of started and ended